### PR TITLE
[G2M] added Text widget icon

### LINKED ISSUE #164: Create new Text Icon
**Issue:**

- we need to add an icon for Text widget type in Widget Studio

Screen Shot 2023-01-20 at 9 36 38 AM

Figma file:
https://www.figma.com/file/DwLE8rXq7MTaVGSkoFKhJE/Widget-Studio?node-id=3001%3A34131&t=iWDnv5T6cn3rVvMQ-4

### PR DESCRIPTION
**Pull request by Hamaad102**
resolves #164 

### Changes
- added new icon to be used for Text widget

Screenshot 2023-01-20 at 9 55 48 AM

### CODE REVIEW

File: src/icons/text.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const iconSize = Object.freeze({
+  lg: 'w-3.5 h-3.5',
+  md: 'w-3 h-3',
+  sm: 'w-2.5, h-2.5',
+})
+
+const Text = ({ className, size, ...props }) => {
+  return (
+    <svg className={`${iconSize[size]} ${className}`} viewBox="0 0 16 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg" {...props}>
```
Comment (geoerika) [inline src/icons/text.js:13]: @Hamaad102 please look at the other files for icons & arrange props & sections similarly.

File: src/icons/text.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </svg>
+
```
Comment (geoerika) [inline src/icons/text.js:16]: no need for empty line here

**Review by geoerika — APPROVED** [review]
Looks good. Please, just arrange a bit the file to look like the other widget icons (ex. PieChart).

**Review by kc-leung — APPROVED** [review]

**Comment by geoerika** [comment]
@Hamaad102  it looks much better. Thank you!

**Comment by geoerika** [comment]
@Hamaad102 don't forget to upgrade version so we can make a new release.

**Pull request merged**